src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_storage.v1.json.baseline

```diff
@@ -6,35 +6,27 @@
 #    https://console.developers.google.com/apis/api/storage
 # 2. This sample uses Application Default Credentials for authentication.
 #    If not already done, install the gcloud CLI from
-#    https://cloud.google.com/sdk/ and run
+#    https://cloud.google.com/sdk and run
 #    `gcloud beta auth application-default login`
 # 3. Install the Ruby client library and Application Default Credentials
 #    library by running `gem install google-api-client` and
 #    `gem install googleauth`
+
 require 'googleauth'
 require 'google/apis/storage_v1'
-require 'json'
 
 service = Google::Apis::StorageV1::StorageService.new
 
-# Use Application Default Credentials for authentication when running locally.
-# For more information, see:
-# https://developers.google.com/identity/protocols/application-default-credentials
 service.authorization = \
     Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
 
-
-# TODO: Change placeholders below to desired parameter values for the `delete_bucket_access_control` method:
-
 # Name of a bucket.
-bucket = '{MY-BUCKET}'
+bucket = ''  # TODO: Update placeholder value.
 
 # The entity holding the permission. Can be user-userId, user-emailAddress, group-groupId,
 # group-emailAddress, allUsers, or allAuthenticatedUsers.
-entity = '{MY-ENTITY}'
-
+entity = ''  # TODO: Update placeholder value.
 
-# Perform the call
```

Comment:
Currently Ruby compile check relies on this comment to segment the fragment files; it was on my TODO list to fix this. Do you want to leave it and open an issue?